PhysicsTools/PatAlgos/python/slimming/primaryVertexAssociation_cfi.py

```diff
@@ -4,10 +4,17 @@
 primaryVertexAssociation = sortedPFPrimaryVertices.clone(
   qualityForPrimary = cms.int32(2),
   produceSortedVertices = cms.bool(False),
-  producePileUpCollection  = cms.bool(False),  
+  producePileUpCollection  = cms.bool(False),
```

Comment:
Please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

```PYTHON
primaryVertexAssociation = sortedPFPrimaryVertices.clone(
  qualityForPrimary = 2,
  produceSortedVertices = False,
  producePileUpCollection  = False,
  produceNoPileUpCollection = False
)
```